README.md

```diff
@@ -232,17 +232,29 @@ cd $GOPATH/src/github.com/kubeedge/kubeedge
 
 ### Build Cloud
 
-```shell
-cd $GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller
-make # or `make edgecontroller`
-```
++ Modify the `$GOPATH/src/github.com/kubeedge/kubeedge/cloud/edgecontroller/conf/ontroller.yaml` configuration file to modify `cloudhub.ca`, `cloudhub.cert`, `cloudhub.key` to generated Certificate path
```

Comment:
ontroller.yaml => controller.yaml

Reply:
Maybe we can keep these 3 places `Certificate name` with consistency.
So that we can avoid the users to modify too many places.
https://github.com/kubeedge/kubeedge#generate-certificates
https://github.com/kubeedge/kubeedge/blob/master/cloud/edgecontroller/conf/controller.yaml#L12-L14
https://github.com/kubeedge/kubeedge/blob/master/edge/conf/edge.yaml#L12-L13
@zhanghongtong any thoughts?

Reply:
Yes, the different certificate names in these three places will lead to confusion for new users.

Reply:
I think they should unify the name and path

Reply:
@zhanghongtong :+1:
Could you unify them? Thanks

Reply:
Of course :）